experimental/opencv-text-recognition/text_recognition.py

```diff
@@ -11,9 +11,15 @@
 from tqdm import tqdm
 from scipy.ndimage import interpolation as inter
 import cv2
+from typing import List
+
```

Comment:
modules should be self-contained, of course `numpy` is imported elsewhere, but it would be better to have this import explicitly here.